base/linalg/generic.jl

```diff
@@ -324,6 +590,12 @@ trace(x::Number) = x
 #det(a::AbstractMatrix)
 
 inv(a::StridedMatrix) = throw(ArgumentError("argument must be a square matrix"))
+
+"""
+    inv(M)
+
+Matrix inverse.
```

Comment:
`inv` docstring - expand with at least the definition, maybe also a few words on how we calculate it?